rasa_nlu/training_data/util.py

```diff
@@ -24,3 +25,23 @@ def check_duplicate_synonym(entity_synonyms, text, syn, context_str=""):
     if text in entity_synonyms and entity_synonyms[text] != syn:
         logger.warning("Found inconsistent entity synonyms while {0}, overwriting {1}->{2}"
                        "with {1}->{2} during merge".format(context_str, text, entity_synonyms[text], syn))
+
+
+def generate_lookup_regex(file_path, print_data_size=True):
```

Comment:
`print_data_size` isn't used anywhere, please remove it

Reply:
ah yes, good catch.  Originally I had this as an optional functionality but now just print to logger regardless.